UnityProject/Assets/Scripts/Items/Implants/Limb.cs

```diff
@@ -7,73 +7,24 @@ namespace HealthV2
 {
 	public class Limb : BodyPartFunctionality, IMovementEffect
```

Comment:
you should be able to remove ``IMovementEffect`` from here and the arm script

Reply:
actually you need the IMovementEffect on the arm script for crawling

Reply:
oh, crawling uses arms speed? Aight kinda cool.

Reply:
you shuold still remove the interface from the base limb class though

Reply:
can't remove it because the base function has this functionality
`playerHealth.OrNull()?.PlayerMove.AddModifier(this);`

Reply:
don't resolve conversations by yourself, please. You have 3 courses of action then:
A - completely remove speed from arms and move this functionality to legs which is the only place it would matter.
B - Remove the interface from the inheritos of limb, they are are IMovementEffect since they inherit from Limb.
C - Remove the interface from Limb and move this method call to both inheritors.

Reply:
done